mobile_app/lib/view/cafe_detail/cafe_detail_body_content.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/material.dart';
+import 'package:mobile_app/constants/type.dart';
+import 'package:mobile_app/util/cafe_detail.dart';
+import 'package:mobile_app/view/cafe_detail/cafe_detail_footer.dart';
+import 'package:mobile_app/view/cafe_detail/cafe_detail_info.dart';
+import 'package:mobile_app/view/cafe_detail/cafe_detail_location.dart';
+import 'package:mobile_app/view/cafe_detail/cafe_detail_menu.dart';
+import 'package:mobile_app/view/common/cafe_image_slider.dart';
+import 'package:mobile_app/view/common/image_index_bullet.dart';
+
+const double FOOTER_HEIGHT = 72;
```

Comment:
클래스는 PascalCase, 값은 camelCase로 변수명을 지정해야 합니다. (see [Effective Dart](https://dart.dev/guides/language/effective-dart/style#do-name-types-using-uppercamelcase))
또한 다른 커멘트와 마찬가지로, 파일 내에서 사용되는 값은 `_` prefix를 붙여 private하게 만들어야 합니다.

```suggestion
const double _footerHeight = 72;
```